coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
it's a path of the directory where all the json files have been stored. Um we need to pass this. I am not sure of any alternative.  Thoughts?
cc @AbdealiJK